docs/continuous_integration.rst

```diff
@@ -1,11 +1,44 @@
 Continuous Integration
 ======================
 
+The "CI" in CumulusCI stands for "continuous integration." Continuous integration is the practice of automatically running a project's tests for any change before that change is merged to the ``main`` branch in the repository. Continuous integration also configures the repository so that changes are merged only if the tests have passed. This practice keeps the ``main`` branch in an error-free state where it can be released any time.
+
+Teams can put in significant effort to create bespoke automation for CumulusCI tailored to their project's needs. When created, the automation is available to all project participants, from Developers and Quality Engineers, to Documentation Writers and Product Managers. CumulusCI takes this reuse of automation one step further by letting it run in the context of CI systems like GitHub Actions, CircleCI, or Azure Pipelines. This consistent reuse of automation from local environments to cloud-based CI systems gives teams the ability to develop, test, and deploy their projects with confidence.
+
+
+
 CumulusCI Flow
 --------------
 
-Run CumulusCI from Github Actions
----------------------------------
+CumulusCI Flow is the process by which Salesforce metadata is developed, tested, and deployed to our customers. It is similar to the GitHub Flow with a few tweaks and additions. To learn more about which CumulusCI flows are best designed for CI builds, branching and merging strategies for repositories, generating product documentation during project initialization, and more, see `CumulusCI Flow<TODO>`_.
```

Comment:
I added more detail to each of these examples. Please confirm that they are correct.